CDCgov/prime-reportstream
# NY UP Onboarding

This PR adds NY receiver transforms needed to onboard NY to the Universal Pipeline

**If you are suggesting a fix for a currently exploitable issue, please disclose the issue to the prime-reportstream team directly outside of GitHub instead of filing a PR, so we may immediately patch the affected systems before a disclosure. See [SECURITY.md/Reporting a Vulnerability](https://github.com/CDCgov/prime-reportstream/blob/master/SECURITY.md#reporting-a-vulnerability) for more information.**

Test Steps:
1. Check out and build the branch
2. run `./prime multiple-settings set -e local -i settings/staging/0155-NY-up-onboarding.yml`
3. Send a test report using the attached Postman collection
[NY.postman_collection.txt](https://github.com/CDCgov/prime-reportstream/files/12541742/NY.postman_collection.txt)
4. Verify that the request was accepted
5. Open the generated HL7 file in the local sftp folder.
6. Verify the file name contains NY receiver transforms specified in `NY-receiver-transforms.yml`

## Changes
- Added NY receiver transforms
- Enabled 6 second offset back to default sender transforms

## Checklist

### Testing
